### PR TITLE
[Form v2] Checkbox

### PR DESCRIPTION
**Pull request by vvalentin-lucca**
## Description

- [x] New documentation
- [x] New .checkbox is named `.checkboxField`.
- [x] Older .checkbox is not renamed.

-----

https://github.com/LuccaSA/lucca-front/assets/25581936/fbf98e6e-38c7-43b9-9684-149225499627

image

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2136/storybook

**Review by RipripKipkip — DISMISSED** [review]

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/4171/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2136/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/4230/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2136/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/4233/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2136/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2136/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2136/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/5249/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2136/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/5261/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2136/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/5263/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2136/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/5264/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2136/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/5265/ui-diff)

File: packages/scss/src/components/checkboxField/component.scss [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+				direction: ltr;
+				font-family: 'Lucca icons';
+				font-feature-settings: 'liga';
+				font-style: normal;
+				letter-spacing: normal;
+				line-height: 1;
+				text-transform: none;
+				white-space: nowrap;
+				word-wrap: normal;
+				content: '\e97d';
```
Comment (BertrandPodevin) [inline packages/scss/src/components/checkboxField/component.scss:68]: add an empty alternative text here ? `content: '\e97d' / '';`

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2136/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/5271/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2136/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/5272/ui-diff)

**Review by BertrandPodevin — APPROVED** [review]

**Pull request merged**